packages/communication-react/package.json

```diff
@@ -89,7 +89,7 @@
     "_by-flavor": "rushx _current-flavor && env-cmd -f ../../common/config/env/.env --use-shell"
   },
   "devDependencies": {
-    "@azure/communication-calling": "1.13.2-beta.1 || >=1.13.1",
+    "@azure/communication-calling": "1.14.1-beta.1 || >=1.13.1",
     "@azure/communication-calling-effects": "1.0.0-beta.2",
     "@azure/communication-chat": "1.3.2-beta.2 || >=1.2.0",
     "@azure/core-auth": "1.3.2",
```

Comment:
Is this one missed or on purpose

Reply:
which "one" are you referring to?  If its about adding a caret, because its a beta version and doesn't follow semver you cant use a caret on it.

Reply:
updated to include @azure/core-auth ^1.4.0

https://www.npmjs.com/package/@azure/core-auth?activeTab=versions